apps/passport-client/components/screens/ProveScreen/GenericProveSection.tsx

```diff
@@ -244,7 +244,7 @@ export function GenericProveSection<T extends PCDPackage = PCDPackage>({
           <Button2
             style={{ marginTop: "auto" }}
             onClick={() => {
-              window.history.back();
+              window.close();
```

Comment:
where does this check for history? wouldn't this always close for cases where the prove screen pops up

Reply:
It was my understanding that the only way to get to the prove screen is from outside of the app, is this not the case?

Reply:
a different site could redirect you to the prove screen, in which case, this logic would close the window rather than going back in browser history